test/backend/certificates.spec.js

```diff
@@ -0,0 +1,88 @@
+import {assert, expect} from 'chai';
+import fs from 'fs';
+import {contains, dissoc, merge} from 'ramda';
+
+import {getSetting, saveSetting} from '../../backend/settings.js';
+import {
+    getCerts,
+    CA_HOST_URL,
+    calculateDaysToRenewal,
+    fetchCertsFromCA,
+    saveCertsLocally,
+    setRenewalJob,
+    renewCertificate
+} from '../../backend/certificates';
+import {username, accessToken} from './utils';
+
+const cleanUp = () => {
+    ['KEY_FILE', 'CERT_FILE'].forEach(fileName => {
+        try {
+            fs.unlinkSync(getSetting(fileName));
+        } catch (e) {}
+    });
+    try {
+        fs.unlinkSync(getSetting('SETTINGS_PATH'));
+    } catch (e) {}
+};
+
+describe('Certificates', function() {
+    beforeEach(() => {
+        cleanUp();
+    });
+
+    afterEach(() => {
+        cleanUp();
+    });
+
+    it('The CA URL is correctly defined.', () => {
+        assert.equal(CA_HOST_URL, 'http://plotly-connector-test.com/certificate');
```

Comment:
Good point. I'll move it to the test fixture to specify a CA_HOST_URL and the prod will have the prod CA_HOST_URL whatever it ends up being i.e. plotly-connector.com